opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -297,6 +297,10 @@ enum AggregationTemporality {
   // systems that do not use start_time to determine when the aggregation
   // value was reset (e.g. Prometheus).
   AGGREGATION_TEMPORALITY_CUMULATIVE = 2;
+
+  // GAUGE is an AggregationTemporality for a metric of type GAUGE. Essentially
+  // it means that aggregation over time is not being used.
+  AGGREGATION_TEMPORALITY_GAUGE = 3;
```

Comment:
This is interesting because it does not apply to other places except where you wanted which is for Histogram to support OpenCensus and OpenMetrics concept of gauge Histograms

Reply:
Generally I think adding a DoubleGaugeHistogram would be a tidier solution, but the lack of it seemed so odd, especially in comparison to the old open census types, that I assumed leaving it out was a deliberate decision. I'm happy to add it, instead of the aggregation_temporality_gauge, if people are ok with a new metric type.

Reply:
I think I like more a new type.

Reply:
I've updated the pull request to instead add a metric type DoubleGaugeHistogram